examples/image_classification/ImageClassificationOpenCVRunner.js

```diff
@@ -0,0 +1,25 @@
+class ImageClassificationOpenCVRunner extends OpenCVRunner {
+  constructor() {
+    super();
+    this._labels = null;
+  }
+
+  _setLabels = (labels) => {
+    this._labels = labels;
+  };
+
+  _getLabels = async (url) => {
+    const result = await this._loadURL(url);
+    this._setLabels(result.split('\n'));
+    console.log(`labels: ${this._labels}`);
+  };
+
+  _getOtherResources = async () => {
+    await this._getLabels(this._currentModelInfo.labelsFile);
+  };
+
+  _updateOutput = (output) => {
```

Comment:
better to name `_postProcess`?

Reply:
Thanks for your suggestion, the method [_processOutput](https://github.com/intel/webml-polyfill/blob/master/examples/util/BaseExample.js#L450) is for post processing with inferenced output.
 And this _updateOutput method is to customize some 'output' from runner to example by the public getOutput method of runner.

Reply:
I am fine with this name. However, there are processOutput, processCustomOutput and updateOutput. Can we document them?

Reply:
OK, got it, thanks.